src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,290 @@
+package duke;
+
+import duke.exceptions.EmptyCommandException;
+import duke.exceptions.UnknownCommandException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+
+    public static final String EXIT_PROGRAM = "bye";
+    public static final String LOGO = "   _____  .__   _____                   .___\n" +
+            "  /  _  \\ |  |_/ ____\\______   ____   __| _/\n" +
+            " /  /_\\  \\|  |\\   __\\\\_  __ \\_/ __ \\ / __ |\n" +
+            "/    |    \\  |_|  |   |  | \\/\\  ___// /_/ |\n" +
+            "\\____|__  /____/__|   |__|    \\___  >____ |\n" +
+            "        \\/                        \\/     \\/\n";
+    public static final String WELCOME_MESSAGE = "Hello from\n";
+    public static final String DIVIDER = "____________________________________________________________\n";
+    public static final String GREETINGS = " Hello! I'm Alfred Pennyworth.\n What can I do for you?\n";
+    public static final String ENDING = " Bye. Hope to see you again soon!\n";
+    public static final String ADDED_TASK = "Got it. I've added this task:\n  ";
+    public static final String BY_DELIMITER = " /by ";
+    public static final String FROM_DELIMITER = " /from ";
+    public static final String TO_DELIMITER = " /to ";
+    public static final String MARKED_THIS_TASK_AS_DONE = "Nice! I've marked this task as done: ";
+    public static final String UNMARKED_THIS_TASK_AS_DONE = "Okay, I've marked this task as not done yet: ";
+    public static final String UNKNOWN_COMAMND_MESSAGE = "OOPS!!! I'm sorry, but I don't know what that means :-(";
+    public static final String DELETED_THIS_TASK = "Noted. I've removed this task:";
+    public static final String MARK_UNMARK_INDEX_IS_NOT_A_NUMBER_MESSAGE = "mark/unmark index is not a number";
+    public static final String MARK_UNMARK_INDEX_DOES_NOT_EXIST_MESSAGE = "mark/unmark index does not exist";
+    public static final String TODO_COMMAND = "todo";
+    public static final String DEADLINE_COMMAND = "deadline";
+    public static final String EVENT_COMMAND = "event";
+    public static final String MARK_COMMAND = "mark";
+    public static final String UNMARK_COMMAND = "unmark";
+    public static final String LIST_COMMAND = "list";
+    public static final String DELETE_COMMAND = "delete";
```

Comment:
Perhaps associated constant names should have a common prefix? Same for the `DELIMITER` constants as well.

https://se-education.org/guides/conventions/java/index.html#variables